lib/services/alexa/request/lifecycle/response.ts

```diff
@@ -15,6 +16,15 @@ const utilsObj = {
   responseHandlers,
 };
 
+const removeHiddenVariables = (state: State): State => ({
+  ...state,
+  variables: {
+    ...state.variables,
+    [V.CONTEXT]: undefined,
+    [V.SYSTEM]: undefined,
+  },
+});
```

Comment:
just to not save these large, unwieldy objects into the database every interaction